integrations/astroport/packages/abstract-adapter/Cargo.toml

```diff
@@ -28,10 +28,6 @@ astroport = { path = "../astroport/", optional = true }
 lazy_static = "1.4.0"
 
 [dev-dependencies]
-abstract-dex-adapter-traits = { version = "0.17.2", features = ["testing"] }
+abstract-dex-adapter-traits = { version = "0.18.0", features = ["testing"], path = "../../../../framework/packages/dex"}
 cw-orch = { version = "0.13", features = ["daemon"] }
 ibc-chain-registry = "0.24.1"
-
-[patch.crates-io]
-abstract-staking-adapter-traits = { path = "../../../../framework/packages/staking" }
-abstract-dex-adapter-traits = { path = "../../../../framework/packages/dex" }
```

Comment:
What's the diff between the patch one and the inline one?

Reply:
For some reason it was ignoring patch one, noticed that we use the inline on main branch as well

Reply:
Nice! Thanks for making the change.